.buildkite/pipeline.yml

```diff
@@ -107,3 +107,70 @@ steps:
         depends_on:
           - "operator-image-build"
           - "e2e-tests-image-build"
+
+  - group: redhat-release
+
+    steps:
+
+      - label: "build operatorhub tool"
+        key: "build-operatorhub-tool"
+        if: build.message == "run operatorhub release"
+        commands:
+           - cd hack/operatorhub
+           - make build
+           - buildkite-agent artifact upload "bin/operator*"
```

Comment:
Would it make sense to use a custom image for this to not have to build the tooling every time?

Reply:
@pebrc the build process takes ~60 seconds in BK.  If we add this binary to the image we're already using, then the process to fix any new Red Hat changes in the release process (which are pretty common) would involve:

1. opening a PR in this repository, waiting for approval/merge of the tooling.
2. opening a PR in the image repository to bump the tooling version, waiting for approval/merge.
3. Re-running the bk job.

As opposed to:
1. opening a PR in this repository with the tooling changes
2. running the bk job, pointing to the open PR for the tooling changes.  (note the `--buildkite-pr-repository` and `--buildkite-pr-id` flags in the `operatorhub buildkite` command which exist for this scenario)